oracle/util_test.go

```diff
@@ -331,7 +331,7 @@ func TestStandardDeviation(t *testing.T) {
 				},
 				OJOUSD: {
 					mean:      sdk.MustNewDecFromStr("1.1335"),
-					deviation: sdk.MustNewDecFromStr("0.004600724580614015"),
+					deviation: sdk.MustNewDecFromStr("0.004600724580614014"),
```

Comment:
Why did this change by .000000000000000001? Rounding change in golang?

Reply:
Could be that or updates to the math functions in the sdk